src/Tizen.TV.UIControls.Forms/Renderer/GridViewRenderer.cs

```diff
@@ -0,0 +1,168 @@
+        IList<GengridItemContext> itemContexts = new List<GengridItemContext>();
+        ElmSharp.GenGrid _genGrid = null;
+
+        ElmSharp.GenItemClass gridItemClass = new ElmSharp.GenItemClass("default")
+        {
+            GetContentHandler = (obj, part) =>
+            {
+                GengridItemContext context = (GengridItemContext)obj;
+
+                if (part == "elm.swallow.icon")
+                {
+                    var renderer = Platform.GetOrCreateRenderer(context.RealizedView);
+
+                    if (renderer is LayoutRenderer)
+                    {
+                        (renderer as LayoutRenderer).RegisterOnLayoutUpdated();
+                    }
+                    return renderer.NativeView;
+                }
+                return null;
+            }
+        };
+
+        protected override void OnElementChanged(ElementChangedEventArgs<GridView> e)
+        {
+            if (Control == null)
+            {
+                _genGrid = new ElmSharp.GenGrid(Xamarin.Forms.Forms.NativeParent)
+                {
+                    HorizontalScrollBarVisiblePolicy = Element.IsHorizontalScrollBarVisible == false ? ElmSharp.ScrollBarVisiblePolicy.Invisible : ScrollBarVisiblePolicy.Visible,
+                    VerticalScrollBarVisiblePolicy = Element.IsHorizontalScrollBarVisible == false ? ElmSharp.ScrollBarVisiblePolicy.Invisible : ScrollBarVisiblePolicy.Visible,
+                    IsHorizontal = Element.IsHorizontal,
+                    ItemWidth = Element.ItemWidth,
+                    ItemHeight = Element.ItemHeight,
+                    ItemAlignmentX = Element.ItemHorizontalAlignment,
+                    ItemAlignmentY = Element.ItemVerticalAlignment,
+                    Style = Element.ThemeStyle,
+                };
+                _genGrid.ItemSelected += OnItemSelected;
+                _genGrid.ItemFocused += OnItemFocused;
+                _genGrid.ItemUnfocused += OnItemUnfocused;
+                
+                if (Element.ItemsSource != null)
+                {
+                    UpdateItemsSource();
+                }
+                SetNativeControl(_genGrid);
+            }
+            base.OnElementChanged(e);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == GridView.ItemWidthProperty.PropertyName)
+            {
+                Control.ItemWidth = Element.ItemWidth;
+            }
+            else if (e.PropertyName == GridView.ItemHeightProperty.PropertyName)
+            {
+                Control.ItemHeight = Element.ItemHeight;
+            }
+            else if (e.PropertyName == GridView.ThemeStyleProperty.PropertyName)
+            {
+                Control.Style = Element.ThemeStyle;
+            }
+            else if (e.PropertyName == GridView.ItemsSourceProperty.PropertyName)
+            {
+                UpdateItemsSource();
+            }

```

Comment:
Why not handle others properties? ex) ScrollBarVisiblePolicy, IsVerticalScrollBarVisible, ThemeStyle

Reply:
I will fix it. Thank you.